Source/player.h

```diff
@@ -395,6 +395,49 @@ struct PlayerStruct {
 	 */
 	void Reset();
 
+	/**
+	 * @brief Return armor value displayed in character sheet
+	 */
+	int GetSheetArmor() const {

```

Comment:
This is used mostly out side of the sheet, maybe GetBaseArmor(). GetArmorRating() or GetArmor() would be better names.

Reply:
probably, I wanted to show that the sheet is not lying anymore xD